### PR TITLE
refactor(axis): use css-classes instead of style

### PR DESCRIPTION
**Pull request by max-jonas**
* adds css-classes to ticks to make them more accessible from outside

### Updates

- issues #1896

### CODE REVIEW

**Comment by max-jonas** [comment]
I have read the DCO document and I hereby sign the DCO.

**Comment by max-jonas** [comment]
recheck

File: packages/core/scss/components/_axis.scss [hunk 1]
```diff
@@ -32,6 +32,14 @@
 		g.tick text {
 			fill: theme.$text-secondary;
 			font-family: var(--#{globals.$prefix}-charts-font-family-condensed);
+
+			&.primary-tick-label {
```
Comment (theiliad) [inline packages/core/scss/components/_axis.scss:36]:
```suggestion
			&.tick-label--primary {
```

File: packages/core/src/components/axes/axis.ts [hunk 2]
```diff
@@ -406,11 +406,11 @@ export class Axis extends Component {
...
 				.order()
 				.select('text')
-			ticks.style('font-weight', (tick: number, i: number) => {
-				return isTickPrimary(tick, i, axis.tickValues(), timeInterval, showDayName)
-					? 'bold'
-					: 'normal'
-			})
+			ticks.attr('class', (tick: number, i: number) =>
+				isTickPrimary(tick, i, axis.tickValues(), timeInterval, showDayName)
+					? 'primary-tick-label'
```
Comment (theiliad) [inline packages/core/src/components/axes/axis.ts:411]:
```suggestion
					? 'tick-label--primary'
```

**Review by theiliad — CHANGES_REQUESTED** [review]

**Review by theiliad — APPROVED** [review]

**Pull request merged**